src/argparse/arguments/FlagInputArgument.java

```diff
@@ -38,14 +59,16 @@ public FlagInputArgument(String token, String alias, String help) {
     @Override
     public boolean resolveArgument(String[] args) {
         for (int index = 0; index < args.length; index++) {
-            if (stringArgEquals(args[index])) {
-                if (index + 1 < args.length && !args[index + 1].substring(0, 1).equals("-")) {
+        	if (stringArgEquals(args[index])) {
```

Comment:
Not sure how but it looks like somehow you inserted an actual tab character instead of 4 spaces? It looks like you used spaces pretty much everywhere else ( which I prefer). Can you make sure there aren't any tab characters?